client/src/app/App.js

```diff
@@ -1092,6 +1096,10 @@ export class App extends PureComponent {
     });
   }
 
+  displayNotification(options) {
+    return this.notifications.current.displayNotification(options);
```

Comment:
Shouldn't we pull out the notification logic out of the component that displays them so we could avoid using `current`.

Reply:
Why should we?